app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -97,7 +99,8 @@ public static void setLayoutMargin(@NonNull View view, float margin) {
     }
   }
 
+  @SuppressLint("WrongConstant")
```

Comment:
Sorry, I think you may have misunderstood my previous comment. I think that this lint error is coming from the fact that we're mixing View.getLayoutDirection and ViewCompat. We need to use VIewCompat for both the layout direction retrieval and the constant.

Reply:
Yeah correct I misunderstood. Now I have fixed it.